chai_lab/data/sources/rdkit.py

```diff
@@ -157,8 +157,8 @@ def generate(self, smiles: str) -> ConformerData:
 
         AllChem.EmbedMultipleConfs(mol_with_hs, numConfs=1, params=params)
         AllChem.RemoveHs(mol_with_hs)
-        for atom in mol_with_hs.GetAtoms():
-            atom.SetProp("name", atom.GetSymbol())
+        for i, atom in enumerate(mol_with_hs.GetAtoms()):
+            atom.SetProp("name", atom.GetSymbol()+str(i+1))
```

Comment:
This a great addition, but I think we can modify this slightly to better conform to prior naming conventions. In particular, instead of numbering every atom in strictly ascending order, let's instead number every atom with as the i-th element.

Something like:

```
element_counter = defaultdict(int)
for i, atom in enumerate(mol_with_hs.GetAtoms()):
    elem = atom.GetSymbol()
    element_counter[elem] += 1
    atom.SetProp("name", elem + str(element_counter[elem]))
```

Reply:
Sorry I made a bit of a mess of it but I think it should pass now.